runtime/compiler/control/DLLMain.cpp

```diff
@@ -221,7 +221,16 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void * reserved)
 #endif//TR_HOST_X86
             )
                {
-               J9VMDllLoadInfo* gcLoadInfo = FIND_DLL_TABLE_ENTRY( J9_GC_DLL_NAME );
+               J9VMDllLoadInfo* gcLoadInfo = NULL;
```

Comment:
Please use the new helper here as it is retrieving the loadInfo.

Reply:
I've having trouble finding a good home for the new helper. If I keep it in `mminit.h`/`mminit.cpp`, it works fine for `runtime/gc/dllinit.c`, but I'm running into the following when I try to include the header and I end up having to link against `gc_modron_startup`:

```
Undefined symbols for architecture x86_64:
  "_MemoryManagerFunctions", referenced from:
      _gcInitializeDefaults in libj9modronstartup.a(mminit.cpp.o)
  "tgcParseArgs(J9JavaVM*, char*)", referenced from:
      _gcParseTGCCommandLine in libj9modronstartup.a(mmparse.cpp.o)
  "tgcInitializeRequestedOptions(J9JavaVM*)", referenced from:
      _gcParseTGCCommandLine in libj9modronstartup.a(mmparse.cpp.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I also thought that [jvminitcommon.c](https://github.com/eclipse/openj9/blob/master/runtime/vm/jvminitcommon.c)/[jvminitcommon.h](https://github.com/eclipse/openj9/blob/master/runtime/vm/jvminitcommon.h) would be a logical spot for the helper, but it looks like I'd have to update the CMakelists for `gc_modron_startup`, `compiler` and `gc` to include/link against `j9vm`, which seems a bit overkill.

Would it be sufficient to use the helper everywhere other than here (`runtime/compiler/control/DLLMain.cpp`)? Or maybe there's another way to do this that I'm not thinking of?

Reply:
This has nothing to do with JVMTI, so certainly not there.

It's no problem to create a new file in util for the helper (gcloadinfo.c perhaps) - util is already linked with everything. The function prototype can go in util_api.h.